### DISCUSSION TITLE
[Drawer] disableBodyScroll unsuccessful - targetElement

### DISCUSSION THREAD

**Discussion by mingbackmountain** [chakra-ui/chakra-ui#2273] [Q&A]
![test7](https://user-images.githubusercontent.com/25886138/96570786-f8c65600-12f4-11eb-885b-458420c37d68.PNG)

![test6](https://user-images.githubusercontent.com/25886138/96570775-f2d07500-12f4-11eb-80fd-1216e078d2a5.PNG)

Hello guys, I have implemented chakra drawer in my website. However, I get this error everytime I click the button to call my drawer. I don't how does this error occur. Can somebody please explain to me why this error happoen or provide some guide to solve this error?

Thank you

**Comment by ljosberinn** [comment]
related issue: https://github.com/chakra-ui/chakra-ui/issues/712

are you on v0.8? then there's a fix included

**Reply by mingbackmountain** [reply]
<img width="266" alt="Screen Shot 2563-10-21 at 17 28 18" src="https://user-images.githubusercontent.com/25886138/96708037-db5fbd80-13c2-11eb-896a-6d2837b981d9.png">
 I use the latest version, but the problem still happen?

**Reply by ljosberinn** [reply]
There's a fix included in the issue mentioned*: https://github.com/chakra-ui/chakra-ui/issues/712#issuecomment-712332346
